requirements.txt

```diff
@@ -1,5 +1,6 @@
 anndata>=0.8,<0.9
 docrep>=0.3.0
+git+https://github.com/theislab/moscot
```

Comment:
This cannot be here, please use just `moscot`.
Re failing tests: think they are coming from `scvelo`, try pinning `pandas<2` here temporarily.